n_serial.c

```diff
@@ -93,9 +89,7 @@ const char *serialNoteTransaction(const char *request, size_t reqLen, char **res
         const char *err = serialChunkedReceive((uint8_t *)(jsonbuf + jsonbufLen), &jsonbufAvailLen, true, (CARD_INTRA_TRANSACTION_TIMEOUT_SEC * 1000), &available);
         if (err) {
             _Free(jsonbuf);
-#ifdef ERRDBG
-            NOTE_C_LOG_ERROR(ERRSTR("error occured during receive", c_iobad));
-#endif
+            NOTE_C_LOG_DEBUG(ERRSTR("error occured during receive", c_iobad));

```

Comment:
@zfields Ping me if you want me to explain why the change to NOTE_C_LOG_DEBUG from NOTE_C_LOG_ERROR. I did that in a few places. Easier to explain verbally.

Reply:
Ha! Okay, sure. You can disregard my previous note, I was working my way down before I saw this message.